data/db.json

```
@@ -0,0 +1,27 @@
+{
+    "users": [
+        {
+            "cpf":1234567899,
+            "nome":"Maguila",
+            "email":"maguilabox@gmail.com",
+            "CEP":"123456-123",
+            "password":"boxeador",
+            "img":""
+
+        }
+
+    ],
+    "products":[
+        {
+            "id":2345,
+            "nome_produto":"hot wheels",
+            "descricao":"",
+            "fotos":[],
+            "preco":"",
+            "data_cadastro":"",
+            "status":"",
+            "contador":""
+        }
+    ]
+    
+}
```

Hoje há um padrão das guidelines de sempre deixar uma newline no fim dos arquivos, o github reclama também huahua
```suggestion
}

```